src/AppContainer.js

```diff
@@ -1,22 +1,89 @@
-import React from "react";
```

Comment:
In term of usage that's not really what I expect from an Overlay component,

I would expect to be able to Overlay any component and this component being transform to an overlay =>
```
<Foo /> // display foo component
<Overlay>
  <Foo/> // display foo component in an overlay
</Overlay>
```

Other problem I see with the implementation
=> I need to register that I exist as an overlay => in this file and in overlay file by adding an id
=> no component composition